ibm_i/tests/test_e2e.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from datadog_checks.ibm_i import IbmICheck
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check, aggregator, instance):
+    aggregator = dd_agent_check(instance)
+    aggregator.assert_service_check("ibm_i.can_connect", IbmICheck.CRITICAL)
```

Comment:
Can we assert the expected tags from the service check too? 
```suggestion
    aggregator.assert_service_check("ibm_i.can_connect", IbmICheck.CRITICAL, tags=[<EXPECTED_TAGS>])
```
An example: https://github.com/DataDog/integrations-core/blob/master/ecs_fargate/tests/test_e2e.py#L13